pkg/logs/sender/config.go

```diff
@@ -89,3 +84,15 @@ func BuildEndpoints() (*client.Endpoints, error) {
 
 	return client.NewEndpoints(main, additionals), nil
 }
+
+func isSetAndNotEmpty(config config.Config, key string) bool {
+	return config.IsSet(key) && len(config.GetString(key)) > 0
+}
+
+//getLogsApiKey provides the dd api key used by the main logs agent sender.
+func getLogsAPIKey(config config.Config) string {
+	if isSetAndNotEmpty(config, "logs_config.logs_api_key") {
```

Comment:
do we have to add the `logs_` prefix ? we are already in the `logs_config` namespace

Reply:
Right, I wanted to emphasise on the difference between this api key and the logs one but I agree, the namespace is enough.

Reply:
Updated